css/css-grid/parsing/display-invalid.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
On Grid Layout suite, we have a subfolder for each spec section (using the anchor of the spec as folder name).
For example this test should go under `grid-model`.